### PR TITLE
added tests for ShoppingCart, CartItem, User, Order, Order Item (repository, service, controller)

### PR DESCRIPTION
**Pull request by DmytroFedoryshyn**

### CODE REVIEW

**Comment by DmytroFedoryshyn** [comment]
Shopping cart and cart item coverage - https://prnt.sc/LqIf74xsVbyq

**Comment by DmytroFedoryshyn** [comment]
User, Order, Order item coverage - https://prnt.sc/2yJwFPGiL683

File: src/main/java/bookstore/repository/shoppingcart/ShoppingCartRepository.java [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
-import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
     @EntityGraph(attributePaths = {"items", "items.book"})
-    ShoppingCart getByUser(@Param("user") User user);
+    ShoppingCart getByUser_Email(String username);
```
Comment (sokimaaa) [inline src/main/java/bookstore/repository/shoppingcart/ShoppingCartRepository.java]: violates java code convention
```suggestion
    ShoppingCart getByUser_Email(String username);
```

File: src/main/java/bookstore/repository/order/OrderRepository.java [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-import bookstore.model.User;
 import java.util.List;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
 
 public interface OrderRepository extends JpaRepository<Order, Long> {
     @EntityGraph(attributePaths = {"orderItems", "user"})
-    List<Order> getAllByUser(User user, Pageable pageable);
+    List<Order> getAllByUser_Email(String email, Pageable pageable);
```
Comment (sokimaaa) [inline src/main/java/bookstore/repository/order/OrderRepository.java]: violates java code convention

**Pull request merged**